Python/Tests/Utilities.Python/PythonVisualStudioApp.cs

```diff
@@ -228,7 +228,43 @@ public PythonVisualStudioApp(IServiceProvider site)
                             System.Threading.Thread.Sleep(500);
                         }
                     }
-                    _testExplorer = new PythonTestExplorer(this, element);
+                    Assert.IsTrue(element != null, "Missing Text Explorer window");
+
+                    AutomationElement searchControl = null;
+                    AutomationElement searchBarElement = null;
+                    if (element != null) {
+                        for (int i = 0; i < 30 && searchControl == null; i++) {
+                            searchControl = Element.FindFirst(
+                                TreeScope.Descendants,
+                                new PropertyCondition(
+                                    AutomationElement.AutomationIdProperty,
+                                    "SearchControl"
```

Comment:
this looks for "SearchControl" across all of the IDE which may run into false positive if someone happened to have another tool window with an automation id just like it.

is there any way you could restrict it to look under an element that you know for sure is test explorer tool window?

Reply:
i'll try that